extensions/designer-news/package.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "designer-news",
+  "title": "Designer News",
+  "description": "See top design stories",
+  "icon": "command-icon.png",
+  "author": "itsnwa",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Designer News",
```

Our guidelines are to have the verb for the title and the service as the subtitle

```suggestion
      "title": "Read Top News",
      "subtitle": "Designer News",
```

---

You're right @mathieudutour, and yes, it's top news :)